server/fleet/activities.go

```diff
@@ -304,15 +304,15 @@ func (a ActivityTypeDeletedSavedQuery) Documentation() (activity string, details
 }`
 }
 
-type ActivityTypeDeletedMultipleSavedQuery struct {
+type ActivityTypeDeletedMultipleSavedQueries struct {
 	IDs []uint `json:"query_ids"`
 }
 
-func (a ActivityTypeDeletedMultipleSavedQuery) ActivityName() string {
-	return "deleted_multiple_saved_query"
+func (a ActivityTypeDeletedMultipleSavedQueries) ActivityName() string {
+	return "deleted_multiple_saved_queries"
```

Comment:
Do we need to change this on the backend? Because existing users will have activities with the old `activity_type` name.

Reply:
Not if you think it would cause any inconvenience to users

Reply:
If there's no need they I wouldn't change the activity type. (We have a premium feature that streams activity JSONs to logging destinations. If users are processing such JSON events then this would be a breaking change.)

Reply:
Ah I see! I'll close this out.

Reply:
But the UI changes make sense :)

Reply:
Agreed! In my original PR I intentionally matched the backend to avoid issues. I was hoping I could make it plural but didn't want to break existing events.

Just so it's clear that I promise I tried to use the plural version since it definitely reads better, IMO. 😛